.gitlab/ci/on-push.yml

```
@@ -466,12 +458,46 @@ slack-notify-nightly-build:
 
 xsiam_server_ga:
   extends:
-    - .test_content_on_xsiam_server_instances_base
+    - .test_content_on_cloud_server_instances_base
+  rules:
+    - if: '$CI_PIPELINE_SOURCE =~ /^(push|contrib)$/ && $CI_COMMIT_BRANCH !~ /^[0-9]{2}\.[0-9]{1,2}\.[0-9]$/'
+    - if: '$NIGHTLY'
+      when: always
+  variables:
+    INSTANCE_ROLE: "XSIAM 1.2"
+    GCS_QUEUE_FILE: "queue-ga"
+    TEST_MACHINES_LIST: "test-machines-ga"
+    GCS_LOCKS_PATH: "gs://xsoar-ci-artifacts/content-locks-xsiam"
+    CLOUD_SERVERS_FILE: "xsiam_servers_path"
+    CLOUD_API_KEYS: $XSIAM_API_KEYS
+    ARTIFACTS_FOLDER: "${ARTIFACTS_FOLDER_MPV2}"
+    GCS_MARKET_BUCKET: "${GCS_MARKET_V2_BUCKET}"
+    ENV_RESULTS_PATH: "${ARTIFACTS_FOLDER_MPV2}/env_results.json"
+    GCS_SOURCE_BUCKET: "$GCS_PRODUCTION_V2_BUCKET"
+    GCS_MACHINES_BUCKET: "marketplace-v2-dist-dev/upload-flow/builds-xsiam"
+    SERVER_TYPE: "CLOUD"
+    MARKETPLACE_NAME: "marketplacev2"
+  needs: ["prepare-testing-bucket-mpv2"]
+
+xsoar_ng_server_ga:
+  extends:
+    - .test_content_on_cloud_server_instances_base
   rules:
     - if: '$CI_PIPELINE_SOURCE =~ /^(push|contrib)$/ && $CI_COMMIT_BRANCH !~ /^[0-9]{2}\.[0-9]{1,2}\.[0-9]$/'
     - if: '$NIGHTLY'
       when: always
   variables:
     INSTANCE_ROLE: "XSIAM 1.2"
     GCS_QUEUE_FILE: "queue-ga"
-    TEST_MACHINES_LIST: "test-machines-ga"
\ No newline at end of file
+    TEST_MACHINES_LIST: "test-machines-ga"
+    GCS_LOCKS_PATH: "gs://xsoar-ci-artifacts/content-locks-xsoar-ng"
+    CLOUD_SERVERS_FILE: "xsoar_ng_servers_path"
+    CLOUD_API_KEYS: $XSOAR_NG_API_KEYS
+    GCS_MARKET_BUCKET: "${GCS_MARKET_V2_BUCKET}"
```

This should be the regular bucket no?
```suggestion
    GCS_MARKET_BUCKET: "${GCS_MARKET_BUCKET}"
```